packages/vscode-extension/README.md

```diff
@@ -99,3 +99,5 @@ SuiteCloud Extension for Visual Studio Code is an open source project. Pull Requ
 
 ## License
 Copyright (c) 2021 Oracle and/or its affiliates The Universal Permissive License (UPL), version 1.0. See [License](/LICENSE.txt) for details.
+
+By installing the SuiteCloud Extension for Visual Studio Code, you are accepting the installation of the SuiteCloud SDK dependency under this [license](https://www.oracle.com/downloads/licenses/oracle-free-license.html).
```

Comment:
I guess this need to be updated with Viktor's proposal of specifying the type of license.

Reply:
@antoniofruci same as above.